docs/src/pages/[platform]/components/table/examples/SpanExample.tsx

```diff
@@ -6,20 +6,32 @@ export const SpanExample = () => (
       <TableRow>
         <TableCell />
         <TableCell />
-        <TableCell colspan="2" />
+        <TableCell
+          // @ts-ignore
+          colspan="2"
```

Comment:
Should this be `colSpan`?

Reply:
`colSpan` is the html prop. this seems to work `colspan={{ colSpan: 2 }}` - I think this is another bug

Reply:
Why not using `colspan={{ colSpan: 2 }}`?

Reply:
@0618 The problem here is that the typing is wrong, it should accept a number not a string, so we don't want to do `colspan={{ colSpan: 2 }}`

Reply:
Added to backlog ticket.